tests/core/installables/test_python_environment.py

```
@@ -0,0 +1,104 @@
+from unittest.mock import patch
+
+import pytest
+
+from cloudai.core import BaseInstaller, PythonEnvironment
+
+
+@pytest.fixture
+def installer(slurm_system) -> BaseInstaller:
+    installer = BaseInstaller(slurm_system)
+    installer.system.install_path.mkdir(parents=True)
+    installer._check_low_thread_environment = lambda threshold=None: False
```

Ruff (0.15.12)

[warning] Unused lambda argument: `threshold`

(ARG005)